code/modules/species/station/alraune.dm

```diff
@@ -141,21 +141,8 @@
 	//They don't have lungs so breathe() will just return. Instead, they breathe through their skin.
 	//This is mostly normal breath code with some tweaks that apply to their particular biology.
 
-	var/datum/gas_mixture/breath = null
-	/// If they're wearing a fully sealed suit, their internals take priority.
-	var/fullysealed = FALSE
-	/// If no sealed suit, internals take priority in low pressure environements.
-	var/environmentalair = FALSE
-
-	if(H.wear_suit && (H.wear_suit.min_pressure_protection = 0) && H.head && (H.head.min_pressure_protection = 0))
-		fullysealed = TRUE
-	else // find out if local gas mixture is enough to override use of internals
-		var/envpressure = H.loc.return_pressure()
-		if(envpressure >= hazard_low_pressure)
-			environmentalair = TRUE
-
-	if(fullysealed || !environmentalair)
-		breath = H.get_breath_from_internal()
+	//just fuck off with this snowflake bullshit about checking if they're sealed off and just test for internals. too complicated.
+	var/datum/gas_mixture/breath = H.get_breath_from_internal()
 
 	if(!breath) //No breath from internals so let's try to get air from our location
```

Comment:
this check and the following code is also redundant, and already processed by the get_breath from internals